setenv.sh

```diff
@@ -1,14 +1,44 @@
 # if running the kdb+tick example, change these to full paths
 # some of the kdb+tick processes will change directory, and these will no longer be valid
 export TORQHOME=${PWD}
-export KDBCONFIG=${TORQHOME}/config
-export KDBCODE=${TORQHOME}/code
-export KDBLOG=${TORQHOME}/logs
-export KDBHTML=${TORQHOME}/html
-export KDBLIB=${TORQHOME}/lib
+export TORQDATA=${PWD}
+export TORQAPPHOME=${PWD}
+export KDBLOG=${TORQDATA}/logs
+export KDBHTML=${TORQDATA}/html
+export KDBLIB=${TORQDATA}/lib
```

Comment:
I don't think `KDBHTML` and `KDBLIB` should be relative to `TORQDATA`, should be relative to `TORQHOME` surely?